Upgrade to TinyMCE 4.9.1

gual opened the pull request:
When enabling [imagetools plugin](https://www.tiny.cloud/docs/plugins/imagetools/) for current version of TinyMCE (4.1.0) I got the error `module definition dependecy not found: tinymce/util/Promise`. Upgrading to TinyMCE 4.9.1 made the plugin and the editor work as expected. Tests results on `OK`.

PS: I followed the guidelines, let me know if there is something else required or something I missed. Thanks!

stephenmcd commented:
#1896 is open with a slightly older version - that PR contained some visual issues that needed to be resolved. Can you confirm these aren't present with this PR, and if they are, resolve them first?

Thanks a lot.

gual commented:
@stephenmcd I checked #1896.

## Fullscreen issue
I wasn't able to reproduce any issue on my branch. I get the same behavior with the current tinymce version as with the new one (the expected one).

Current: 
<img width="1443" alt="anadir_publicacion___mezzanine_and_upgrade_to_tinymce_4_9_1_by_gual_ _pull_request__1906_ _stephenmcd_mezzanine" src="https://user-images.githubusercontent.com/1650614/49952933-a5964780-fec2-11e8-8f58-98117aafc4d2.png">

Updated:
<img width="1433" alt="anadir_publicacion___mezzanine" src="https://user-images.githubusercontent.com/1650614/49953012-d4acb900-fec2-11e8-8685-4c2656554097.png">

## Styling issues
I did had the same issue of the editor blowing out into the left-hand column where field names were listed. The reason behind this change was that the new default value for width was `100%` and previously it had `auto`. I added the width option explicit on the default setup and voila...

<img width="1429" alt="anadir_publicacion___mezzanine" src="https://user-images.githubusercontent.com/1650614/49953742-649f3280-fec4-11e8-8b03-c51e9b6eba70.png">

gual commented:
As for the

## Front-end Editing
I get the same behavior with the new version.

Current:
<img width="1408" alt="blog___funde" src="https://user-images.githubusercontent.com/1650614/49954541-518d6200-fec6-11e8-8572-89e926e708e3.png">

Updated:
<img width="1425" alt="blog___funde" src="https://user-images.githubusercontent.com/1650614/49954639-86011e00-fec6-11e8-85a2-e94c7cf33fc3.png">
